extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageProtoUtil.java

```diff
@@ -50,6 +50,13 @@ public static ByteBuffer toByteBuffer(final RowSet rowSet) {
         }
     }
 
+    public static byte[] toByteArray(final RowSet rowSet) {
```

Comment:
Are you sure you want to deviate from using the `ExternalizableRowSetUtils` ser/deser impl?

Reply:
Oops, I misread the call.